<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelInformation</class>
 <widget class="QMainWindow" name="DelInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>删除信息</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="DelGradeButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除班级</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DelStuButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除学生</string>
    </property>
   </widget>
   <widget class="QComboBox" name="GradeComboBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="StucomboBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择班号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="Stulabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择学号：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SureButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>330</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确认删除</string>
    </property>
   </widget>
   <widget class="QLabel" name="SureLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 17, 17);</string>
    </property>
    <property name="text">
     <string>请再次确认，信息删除后无法恢复！</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AgainSureButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确认删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>480</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>取消删除</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
